app/uk/gov/hmrc/incometaxsubscriptionselfemployedfrontend/controllers/agent/BusinessListCYAController.scala

```diff
@@ -30,41 +30,53 @@ import uk.gov.hmrc.incometaxsubscriptionselfemployedfrontend.forms.agent.AddAnot
 import uk.gov.hmrc.incometaxsubscriptionselfemployedfrontend.models.{AddAnotherBusinessModel, No, SelfEmploymentData, Yes}
 import uk.gov.hmrc.incometaxsubscriptionselfemployedfrontend.services.AuthService
 import uk.gov.hmrc.incometaxsubscriptionselfemployedfrontend.utilities.ImplicitDateFormatterImpl
-import uk.gov.hmrc.incometaxsubscriptionselfemployedfrontend.views.html.agent.check_your_answers
+import uk.gov.hmrc.incometaxsubscriptionselfemployedfrontend.views.html.agent.CheckYourAnswers
 import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendController
 
 import javax.inject.{Inject, Singleton}
 import scala.concurrent.ExecutionContext
 
 @Singleton
 class BusinessListCYAController @Inject()(authService: AuthService,
+                                          checkYourAnswers: CheckYourAnswers,
                                           val incomeTaxSubscriptionConnector: IncomeTaxSubscriptionConnector,
                                           mcc: MessagesControllerComponents)
-                                         (implicit val ec: ExecutionContext,
-                                          val appConfig: AppConfig,
+                                         (implicit val appConfig: AppConfig,
+                                          val ec: ExecutionContext,
                                           dateFormatter: ImplicitDateFormatterImpl)
   extends FrontendController(mcc) with I18nSupport with ReferenceRetrieval {
 
   def view(addAnotherBusinessForm: Form[AddAnotherBusinessModel], businesses: Seq[SelfEmploymentData])(implicit request: Request[AnyContent]): Html = {
-    check_your_answers(
-      addAnotherBusinessForm = addAnotherBusinessForm,
-      answers = businesses,
-      postAction = routes.BusinessListCYAController.submit(),
-      implicitDateFormatter = dateFormatter
+    val appendable = checkYourAnswers(
+      addAnotherBusinessForm,
+      businesses,
+      routes.BusinessListCYAController.submit(),
+      dateFormatter
     )
+    appendable
   }
 
+  private val businesses1: Int = appConfig.limitOnNumberOfBusinesses
```

Comment:
It would be good to rename `businesses1` to something more meaningful.

Reply:
It might be better still if I just put those back inline, as they were.